### ISSUE TITLE
Duplicate access keys in the menu bar

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#11094]
### Steps to reproduce

Observe the access keys assigned for the menus in the menu bar.

### Expected behavior

There should be no duplicate access keys

### Actual behavior

Edit, Effects, and Extra all have the access key E.
Tracks and Tools both have the access key T.

### Audacity Version

16902f6

### Operating system

Windows
